.changeset/pretty-dingos-travel.md

```diff
@@ -0,0 +1,14 @@
+---
+"@frontity/components": patch
+"@frontity/comscore-analytics": patch
+"@frontity/connect": patch
+"@frontity/core": minor
+"frontity": patch
+"@frontity/google-ad-manager": patch
+"@frontity/google-analytics": patch
+"@frontity/google-tag-manager-analytics": patch
+"@frontity/html2react": patch
+"@frontity/smart-adserver": patch
```

Comment:
You can remove all the packages from the changeset except for `@frontity/core` (discussion: https://github.com/frontity/frontity/pull/661#discussion_r556687124).

Reply:
Oh, even better! 😄  Thank you!

Reply:
Done!